.gitignore

```diff
@@ -10,6 +10,7 @@ npm-debug.log*
 # Build artifacts
 /src/**/*.js
 /src/**/*.d.ts
+/lib/
```

Comment:
guessing the two above aren't needed anymore?

Reply:
Yeah, I am keeping them for "backwards problems". If we remove them now, existing contributors that have built the repo at least once may commit them by mistake. This isn't required but it could be handy to keep for a while😄